mcs/class/Mono.Profiler.Log/Makefile

```diff
@@ -0,0 +1,12 @@
+thisdir = class/Mono.Profiler
+include ../../build/rules.make
+
+LIBRARY = Mono.Profiler.Log.dll
+LIBRARY_SNK = ../mono.snk
+
+LIB_REFS = System System.Core
+LIB_MCS_FLAGS = /unsafe /keyfile:$(LIBRARY_SNK) /publicsign
+
+NO_TEST = yes
```

Comment:
There will be a whole test suite based on this library that will replace the current profiler test suite. That'll be a separate PR.